js/audio-controls-view.js

```diff
@@ -76,7 +72,7 @@ define(function(require) {
 
         postRender: function() {
           // Add inview listener on entire element
-          $('.'+this.elementId).on('inview', _.bind(this.inview, this));
+          this.$('.audio-inner').on('inview', _.bind(this.inview, this));
```

Comment:
It's caused more issues changing it, so I suggest it is changed back to how it was. Until a better way of perhaps measuring the percentage of the component inview is tried and fully tested I think it's best reverting to how it was. Adding inview to the entire element was used to fix another bug with hidden elements but this has recently been fixed another way #80